run-lstm-with-bmi.py

```diff
@@ -26,7 +26,10 @@
                         list(sample_data['temperature'][3].data)):
     model.set_value('atmosphere_water__liquid_equivalent_precipitation_rate',precip)
     model.set_value('land_surface_air__temperature',temp)
-    print('the temperature and precipitation are set to {:.2f} and {:.2f}'.format(model.temperature, model.total_precipitation))
+    print('the temperature and precipitation set values are {:.2f} and {:.2f}'.format(model.temperature, model.total_precipitation))
```

Comment:
These should be get_values, instead of attributes of the model:

    print('the temperature and precipitation are set to {:.2f} and {:.2f}'.format(model.get_value('land_surface_air__temperature'), 
                                                     model.get_value('atmosphere_water__time_integral_of_precipitation_mass_flux'))